### PR TITLE
Reactathon 2018 - Fundamentals Conf Speaker Videos

### PR DESCRIPTION
**Pull request by dunphyben**
Add videos from Day 1 - Fundamentals - from Reactathon 2018

### CODE REVIEW

File: src/pages/talks/2018-03-an-ios-developers-opinion-of-react-native.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          <a href="https://www.launchdrawer.com" target="_blank">
+            LaunchDrawer
+          </a>
+        </div>
+      ),
+      videoUrl: 'https://www.youtube.com/embed/6anak6P_3Z8',
+      content: (
+        <div>
+          <p>
+            The promise of writing code for iOS and having it “just work” out of the box on Android is a tempting one. That along with hot reloading code and the ability to push over the air updates to published apps make React Native a tempting proposition. This sounds too good to be true so what’s the catch? React Native is a framework for building native apps using Javascript. Many native developers rightly associate hybrid Javascript apps with disappointing consumer experiences and frustrating development environments. Over the years the same promises have been made by many other Javascript based frameworks, so why is React Native any different? This talk will be from the perspective of an experienced iOS developer who has been working with React Native for the last 2 years and has shipped several cross platform apps with it.
```
Comment (romelperez) [inline src/pages/talks/2018-03-an-ios-developers-opinion-of-react-native.js]: Run `npm run format` to prettify these lines.

Comment (dunphyben) [reply src/pages/talks/2018-03-an-ios-developers-opinion-of-react-native.js]: good call

File: src/pages/talks/2018-03-the-client-side-revolution.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+          <p>
+            Building websites has changed dramatically over the last few years.
+            In this session we’ll go on a history trip to discover where we came
+            from, why React is a such a big deal, and where we might be going
+            soon. We’ll discuss what a website is and why we use the tools we do
+            to build them and how this is the best of all times to be building
+            for the web.
+          </p>
+          <p>Presented at Reactathon 2018 in San Francisco.</p>
+          <p style={{ fontSize: 14 }}>
```
Comment (romelperez) [inline src/pages/talks/2018-03-the-client-side-revolution.js]: Maybe using "<small>" would be better, if not I think it's ok.

Comment (dunphyben) [reply src/pages/talks/2018-03-the-client-side-revolution.js]: For styles? I think this is pretty standard for JSX, but then again I'm not familiar with your CSS in JS format. 

It's a single inline element so I think we'll be fine.

Comment (romelperez) [reply src/pages/talks/2018-03-the-client-side-revolution.js]: Sorry, typed incorrectly. I mean, using small tags instead of p with font-size 14px.

Comment (dunphyben) [reply src/pages/talks/2018-03-the-client-side-revolution.js]: I think the `small` tag is too small. I'll just stick with 14px.

**Pull request merged**